cli/options/src/lib.rs

```diff
@@ -113,6 +113,17 @@ impl NormalizePaths for PathOrDash {
     }
 }
 
+#[derive(JsonSchema, Parser, Debug, Clone, Serialize, Deserialize)]
+pub struct ProVerifOptions {
+    #[arg(
+        long,
+        value_parser = parse_inclusion_clause,
+        value_delimiter = ' ',
+        allow_hyphen_values(true)
+    )]
+    assume_items: Vec<InclusionClause>,
```

Comment:
We should add some documentation on that, something in the lines of the `-i` flag or the `FStarOptions`'s `interfaces`